packages/sn-document-viewer-react/src/components/document-widgets/ToggleComments.tsx

```diff
@@ -11,10 +11,10 @@ export const ToggleCommentsWidget: React.FC = () => {
   const viewerState = useViewerState()
   return (
     <ToggleBase
-      isVisible={viewerState.showComments}
+      isVisible={false}
```

Comment:
With this change we lose the ability to close the comments section. Is that something that you want?